v20.1/create-table.md

```diff
@@ -154,13 +161,8 @@ In this example, we create secondary and inverted indexes during table creation.
         type STRING,
         owner_id UUID,
         creation_time TIMESTAMP,
-        status STRING,
-        current_location STRING,
-        ext JSONB,
         CONSTRAINT "primary" PRIMARY KEY (city ASC, id ASC),
         INDEX vehicles_auto_index_fk_city_ref_users (city ASC, owner_id ASC),
-        INVERTED INDEX ix_vehicle_ext (ext),
```

Comment:
The text says "we create secondary and inverted indexes", but this appears to remove the inverted index creation.  Is this intentional?  If so, does the surrounding text need to be edited?